### ISSUE TITLE
Some features don't work with `use_native`

### ISSUE THREAD

**Issue by lpender** [verlok/lazyload#448]
The `callback_enter` doesn't fire when used in tandem with `use_native`.

I believe this was working with 12.5.1, but it may just be that the `use_native` wasn't functioning at that time.

`data-bg-multi` also does not seem to work with `use_native`

This may just be a limitation of the project, if so, just worth noting.

Thank you for this plugin ! 🙏 ! 🙏 ! 🙏

**Comment by verlok** [comment]
Hey @lpender ,
when you set the `use_native` option to `true`, you're saying to the LazyLoad library "If the browser supports native lazy loading, use that". 
Now native lazy loading delegates everything to the browser engine and not to javascript, and the implementation of native lazy loading doesn't apply any class or triggers any callback, so the behaviour you're reporting is expected and correct.
Long story short: if you need to use callbacks and classes, you should not use the `use_native` option.
